cppapi/client/DeviceAttribute.h

```diff
@@ -1301,6 +1301,7 @@ protected :
 	bool check_for_data();
 	bool check_wrong_type_exception();
 	int  check_set_value_size(int seq_length);
+	void init_common_class_members(const char * name,int dim_x,int dim_y);
```

Comment:
@bourtemb Why not make it private?

Reply:
I have nothing against making it private.
DeviceAttributeHistory is the only class inheriting from DeviceAttribute and this class does not need to use this new method and the other classes inheriting from it in the future will probably not need it either because they ill probably call DeviceAttribute constructors directly in their constructors...
We can make it private.